qutip/utilities.py

```diff
@@ -161,29 +161,29 @@ def clebsch(j1, j2, j3, m1, m2, m3):
     vmin = int(np.max([-j1 + j2 + m3, -j1 + m1, 0]))
     vmax = int(np.min([j2 + j3 + m1, j3 - j1 + j2, j3 + m3]))
 
-    c_factor = np.zeros((j1 + j2 + j3 + 1), np.int32)
-    _factorial_prod(j3 + j1 - j2, c_factor)
-    _factorial_prod(j3 - j1 + j2, c_factor)
-    _factorial_prod(j1 + j2 - j3, c_factor)
-    _factorial_prod(j3 + m3, c_factor)
-    _factorial_prod(j3 - m3, c_factor)
-    _factorial_div(j1 + j2 + j3 + 1, c_factor)
-    _factorial_div(j1 - m1, c_factor)
-    _factorial_div(j1 + m1, c_factor)
-    _factorial_div(j2 - m2, c_factor)
-    _factorial_div(j2 + m2, c_factor)
+    c_factor = np.zeros((int(j1 + j2 + j3 + 1)), np.int32)
```

Comment:
Wouldn't it have been easier to add the `int` inside the `_factorial_prod` and `_factorial_div` functions.

Reply:
Yes it would have.